packages/documentation/.storybook/styles/components/typo.scss

```
@@ -30,7 +30,7 @@ body {
     }
 
     .docs-story > :first-child {
-      font-family: tokens.get('post-body', 'font-family');
+      font-family: tokens.get('post-body-font-family');
```

This PR would be a good opportunity to remove the "post-" prefix, don't you think?
```suggestion
      font-family: tokens.get('body-font-family');
```

---

Right now the post- prefix is optional. By removing it do you mean just removing it from the method calls using it right now?

---

@alizedebray This has been implemented as you suggested it now 😉